Make `@astrojs/markdown-remark` browser-safe (#9738)
withastro/astro: .changeset/thick-carrots-run.md

---
"@astrojs/markdown-remark": minor
---

> Uses subpath imports to allow the package to be used in browser environments

Reviewer: ```suggestion
Fixes usage in browser environments by using subpath imports
```
